### ISSUE TITLE
1.42.6 crashes on launch

### ISSUE THREAD

**Issue by matt-downs** [ForkIssues/TrackerWin#595]
Just updated my local Fork instance to version 1.42.6 (from ~1.41.x) and now I am unable to open the app - it crashes on launch after the main window hangs for ~2 seconds. 

I have completely wiped the app and settings from `AppData\Local\Fork` and tried reinstalling. I have also tried pointing the source directory to an empty folder to ensure the issue wasn't something to do with one of my git projects being malformed. No luck though, the app still crashes. 

I'm running Windows 10 Pro version 1803. 

Contents of the log at `AppData\Local\Fork\logs` doesn't appear to be very descriptive: 
```
D 2019-12-16 13:07:15.540 Loading settings.
I 2019-12-16 13:07:15.652 Importing repositories from old format
I 2019-12-16 13:07:15.652 Importing repositories from old format
I 2019-12-16 13:07:15.670 Start ipc server Fork_58144_Pipe_AskPass
D 2019-12-16 13:07:15.670 Waiting for next cross-process event '725'
I 2019-12-16 13:07:15.670 Start ipc server Fork_58144_Pipe_Default
D 2019-12-16 13:07:15.670 Waiting for next cross-process event '725'
I 2019-12-16 13:07:16.385 Fork 1.42.6.0 at C:\Users\downsm\AppData\Local\Fork\app-1.42.6\Fork.exe
I 2019-12-16 13:07:16.385 Git Location: C:\Users\downsm\AppData\Local\Fork\gitInstance\2.24.1\bin\git.exe
I 2019-12-16 13:07:16.385 Log target: AppData log file
I 2019-12-16 13:07:16.537 Last update check: 16/11/2019 12:00:00 AM. Next one: 17/11/2019 12:00:00 AM
I 2019-12-16 13:07:16.761 Restoring 0 sessions.
D 2019-12-16 13:07:16.822 Saving settings.
I 2019-12-16 13:07:16.847 Main: 316, Window: 1338, WindowLoaded: 375, UiReady: 88, Total: 1801
D 2019-12-16 13:07:20.599     4055 ms: Paddle Refresh
```

**Comment by DanPristupov** [comment]
I think this can be a related issue https://github.com/ForkIssues/TrackerWin/issues/565. Could you please contact me by email: winsupport@forkapp.io

**Comment by DanPristupov** [comment]
This crash should be fixed in Fork 1.44

**Comment by matt-downs** [comment]
Yep, all fixed with 1.44! 

Thanks for your help @DanPristupov!
